# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/DOMTypes.ipdlh
# Commit: 40b61344f0da
# Full Hash: 40b61344f0da14f9700a75a8a1c67458e440b68a
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 1f - Implement BrowsingContext::LoadURI. r=nika
#   
#   In order to do cross-process targeted window.open() and link click operations,
#   we need a way to load URIs in the current DocShell of a BrowsingContext,
#   whichever process it lives in.
# ==============================================================================

diff -r 3bffe760e2db -r 40b61344f0da dom/ipc/DOMTypes.ipdlh
--- a/dom/ipc/DOMTypes.ipdlh	Thu Aug 01 16:36:32 2019 -0700
+++ b/dom/ipc/DOMTypes.ipdlh	Thu Aug 01 18:00:32 2019 -0700
@@ -5,6 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 include "mozilla/GfxMessageUtils.h";
+include "mozilla/dom/DocShellMessageUtils.h";
 include "mozilla/layers/LayersMessageUtils.h";
 
 include IPCBlob;